src/VirtualMonoRepo/source-mappings.json

```diff
@@ -71,11 +56,8 @@
             "name": "aspnetcore",
             "defaultRemote": "https://github.com/dotnet/aspnetcore",
             "exclude": [
-                "src/submodules/MessagePack-CSharp/**/*.dll",
-                "**/samples/**/jquery-validation-unobtrusive/.bower.json",
-                "**/samples/**/jquery-validation-unobtrusive/*.js",
-                "src/Installers/Windows/AspNetCoreModule-Setup/*",
-                "src/SignalR/clients/java/signalr/gradle/wrapper/gradle-wrapper.jar"
+                // Non-OSS license - https://github.com/dotnet/source-build/issues/3537
```

Comment:
This issue is closed, but I expect that we will need to do something about this to enable full product build on Windows. Is that work tracked somewhere? It would be better to include link to the active issue.

Reply:
This is likely going to be somewhere under https://github.com/dotnet/aspnetcore/issues/52884 or https://github.com/dotnet/source-build/issues/4022

Yes, we will need these files in some manner to build aspentcore.